mmdet/core/mask/structures.py

```diff
@@ -490,6 +509,28 @@ class PolygonMasks(BaseInstanceMasks):
             compose the object, the third level to the poly coordinates
         height (int): height of masks
         width (int): width of masks
+
+    Example:
+        >>> from mmdet.core.mask.structures import *  # NOQA
+        >>> masks = [
+        >>>     [ np.array([0, 0, 10, 0, 10, 10., 0, 10, 0, 0]) ]
+        >>> ]
+        >>> height, width = 16, 16
+        >>> self = PolygonMasks(masks, height, width)
+
+        >>> # demo translate
```

Comment:
We may also add this to BitmapMasks.

Reply:
I don't want to add too many examples of method calls to the class-level docstring. One or two common use-cases is usually ok. When possible its better if each method has its own docstring and example 

To facilitate this I like to make a `@classmethod` called random, so creating a test instance to play with is a 1-liner. I added those random classmethods to BitmapMask and Polygon (fortunately I've written code for random polygons before, because its somewhat complex). I think I'm starting to go beyond the scope of this PR, so I don't want to add too much more. We can address any concerns with existing code, but for new code we should do a new PR.

Reply:
Agree.